### ISSUE TITLE
pydantic error...

### ISSUE THREAD

**Issue by pe78420** [robberwick/pylibrelinkup#45]
Hi,
When attempting to execute the following: `data = client.graph(patient_identifier=patient)`
I encountered a persistent error: `pydantic_core._pydantic_core.ValidationError: 2 validation errors for GraphResponse`
Could you please help me understand what might be causing this issue?
For reference, the following command works without any error: `data = client.logbook(patient_identifier=patient)`
Many thanks in advance for your assistance.
Best regards,
Philippe

**Comment by robberwick** [comment]
Hi @pe78420 

My guess is that there's something unexpected with something in the graph response payload. Can you post the full content of the error message? that will definitely help to home in on the issue.

**Comment by pe78420** [comment]
Hi @robberwick 
Thanks for the update, below is the full content of the error message:

`Validation Error: 2 validation errors for GraphResponse
data.connection.patientDevice.u
  Field required [type=missing, input_value={'did': '34d70a65-9dc1-11...'fixedLowThreshold': 70}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/missing
data.activeSensors.0.device.u
  Field required [type=missing, input_value={'did': '34d70a65-9dc1-11...'fixedLowThreshold': 70}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/missing`

Let me know if you need any additional details!
Best regards,
Philippe

**Comment by robberwick** [comment]
Hi @pe78420 - do you want to give this branch a try and see if it solves your problem?

```
pip uninstall pylibrelinkup
python -m pip install git+https://github.com/robberwick/pylibrelinkup.git@defaults
```

if it all goes well, i'll merge that branch and prep a release.

**Comment by pe78420** [comment]
Hi @robberwick 
I tested the pylibrelinkup-0.6.2.dev2+gabc2d74 branch, and it seems the issue with Pydantic has been resolved. I was finally able to retrieve the graph data successfully.
Great work!
Philippe

**Comment by robberwick** [comment]
No problem - I've merged that branch and pushed a new release ([v0.6.2](https://pypi.org/project/pylibrelinkup/0.6.2/)), so you should now be able to uninstall that dev branch version and go back to the upstream release:

```
pip uninstall pylibrelinkup
pip install pylibrelinkup
```

**Comment by pe78420** [comment]
Done...
Thanks! 😎
